blockchain/block/block_deserializer.go

```diff
@@ -92,3 +92,35 @@ func (bd *Deserializer) DeserializeBody(buf []byte) (*Body, error) {
 	}
 	return &b, nil
 }
+
+// FromBlockStoreProto converts protobuf to block store
+func (bd *Deserializer) FromBlockStoreProto(pb *iotextypes.BlockStore) (*Store, error) {
+	in := &Store{}
+	blk, err := bd.FromBlockProto(pb.Block)
+	if err != nil {
+		return nil, err
+	}
+	// verify merkle root can match after deserialize
+	if err := blk.VerifyTxRoot(); err != nil {
+		return nil, err
+	}
+
+	in.Block = blk
+	in.Receipts = []*action.Receipt{}
```

Comment:
this line not needed, `in.Receipts` is nil and will be allocated in the `append()` below